app/src/main/res/values-bg/strings.xml

```diff
@@ -256,8 +256,7 @@
   <string name="syncthing_active">Syncthing работи</string>
   <string name="syncthing_disabled">Syncthing не работи</string>
   <!--Toast shown if folder observer fails to traverse a folder-->
-  <string name="toast_folder_observer_stack_overflow">Твърде много папки. Проверяване за cyclic symlinks</string>
-  <!--Toast shown if syncthing failed to create a config-->
+    <!--Toast shown if syncthing failed to create a config-->
```

Comment:
This is incorrectly indented, in all files

I still don't understand why you keep messing around with auto-generated. Just change the english strings and leave the others around which will get updated automatically.
PLease

Reply:
oh sh*** I forgot to set the indent for this project in android studio. So automatically done yes, but incorrectly configured tool.